libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -43,14 +45,12 @@
         </div>
       </div>
     </div>
-  </div>
+  </section>
 </ng-container>
 <ng-template #empty>
   <div class="empty">
-    <p>
-      Try searching for a topic, for example "<a (click)="searchExample()"
-        >JavaScript</a
-      >".
+    <p>Try searching for a topic, for example "
+      <a (click)="searchExample()">JavaScript</a>".
```

Comment:
Is an anchor tag appropriate for this?

Reply:
Yes, an anchor tag is appropriate for this as we can use the anchor tag for click event without a href attribute. Since the anchor tag is an inline element moving down won’t change anything on the UI. We can also use the button element here instead of anchor and style it to look like a href.

Reply:
Accessibility-wise, an anchor tag is _not_ appropriate here. We should be using a button. At the very least we should be using a `role=button` attribute.